src/js/node/tls.ts

```diff
@@ -330,6 +330,7 @@ const TLSSocket = (function (InternalTLSSocket) {
     #socket;
     #checkServerIdentity;
     #session;
+    alpnProtocol = null;
```

Comment:
Does node default to null here? If you want to make sure the property is declared, it is better to use undefined I think.

Reply:
Answer: yes, `null`:

https://github.com/nodejs/node/blob/2eff28fb7a93d3f672f80b582f664a7c701569fb/lib/_tls_wrap.js#L533-L546